package.json

```diff
@@ -351,8 +351,8 @@
     "regenerator-runtime": "^0.13.3"
```

Comment:
update version of app

Reply:
Can you be specific? What version and why? Did you update any packages recently?

Reply:
I mean ui-organizations

Reply:
Oh it's already updated